build-system/tasks/css.js

```diff
@@ -59,6 +59,11 @@ const cssEntryPoints = [
     outJs: 'amp-story-player.css.js',
     outCss: 'amp-story-player-v0.css',
   },
+  {
+    path: 'amp-story-player-iframe.css',
+    outJs: 'amp-story-player-iframe.css.js',
+    outCss: 'amp-story-player-iframe-v0.css',
```

Comment:
Does this match the name of the JavaScript bundle?

Reply:
Actually nvm, I got confused with the other CSS file that publishers have to import.

Reply:
I also got confused by this; should we add comment(s) distinguishing them?

Reply:
Done.